.buildkite/runtest.sh

```diff
@@ -0,0 +1,19 @@
+#!/bin/bash
+
+set -euo pipefail
+
+function download {
+	buildkite-agent artifact download "$1" "$2"
+}
+
+if [ -z "${BUILDKITE}" ]; then
+	echo "This script doesn't appear to be running in buildkite" 1>&2
+	echo "refusing to continue" 1>&2
+	exit 1
+fi
+
+download quark-test .
+chmod +x quark-test
```

Comment:
I'm not sure if this is needed, I think it will keep the permissions from the upload, but it doesn't hurt to have

Reply:
Seems to be, I did a quick test and it failed without.